packages/card/src/ux-card.css

```diff
@@ -5,12 +5,13 @@
   box-sizing: border-box;
   padding: 0;
   border-radius: 2px;
-  background-color: #fff;
-  background-color: var(--aurelia-ux--card-background, var(--aurelia-ux--design-control-background, #fff));
+  background-color: #F5F5F5;
+  background-color: var(--aurelia-ux--card-background, var(--aurelia-ux--design-surface-background, #F5F5F5));
```

Comment:
why is it renamed to surface?

Reply:
To follow material design guideline. See: 
https://material.io/design/color/the-color-system.html#color-theme-creation

And read: 

* The baseline Material color theme
* Surface, background, and error colors

The color theme have a surface color definition (both background and foreground) and it states:

> Surface colors affect surfaces of components, such as cards, sheets, and menus.